docs/collections/documents/tutorials/using_rest_api.md

```diff
@@ -510,221 +666,233 @@ print("Subscription deleted: ", resp.text)
+    this._url = url;
+  }
 
-    login(email, password) {
-      const endpoint = "/_open/auth";
+  login (email, password) {
+    const endpoint = "/_open/auth";
 
-      const self = this;
+    const self = this;
 
-      return new Promise(function (resolve, reject) {
-        self
-          .req(endpoint, {
-            body: { email, password },
-            method: "POST",
-          })
-          .then(({ jwt, ...data }) => {
-            self._headers.authorization = `bearer ${jwt}`;
-            resolve(data);
-          })
-          .catch(reject);
-      });
-    }
+    return new Promise(function (resolve, reject) {
+      self
+        .req(endpoint, {
+          body: { email, password },
+          method: "POST"
+        })
+        .then(({ jwt, ...data }) => {
+          self._headers.authorization = `bearer ${jwt}`;
+          resolve(data);
+        })
+        .catch(reject);
+    });
+  }
 
-    _handleResponse(response, resolve, reject) {
-      if (response.ok) {
-        resolve(response.json());
-      } else {
-        reject(response);
-      }
+  _handleResponse (response, resolve, reject) {
+    if (response.ok) {
+      resolve(response.json());
+    } else {
+      reject(response);
     }
+  }
 
-    req(endpoint, { body, ...options } = {}) {
-      const self = this;
-      return new Promise(function (resolve, reject) {
-        fetch(self._url + endpoint, {
-          headers: self._headers,
-          body: body ? JSON.stringify(body) : undefined,
-          ...options,
-        }).then((response) => self._handleResponse(response, resolve, reject));
-      });
-    }
+  req (endpoint, { body, ...options } = {}) {
+    const self = this;
+    return new Promise(function (resolve, reject) {
+      fetch(self._url + endpoint, {
+        headers: self._headers,
+        body: body ? JSON.stringify(body) : undefined,
+        ...options
+      }).then((response) => self._handleResponse(response, resolve, reject));
+    });
   }
+}
 
-  const EMAIL = "nemo@nautilus.com";
-  const PASSWORD = "xxxxxx";
-  const FEDERATION_NAME = "https://api-gdn.paas.macrometa.io";
-  const FEDERATION_URL = `https://${FEDERATION_NAME}`;
+const email = "nemo@nautilus.com";
+const password = "xxxxxx";
+const federationName = "api-gdn.paas.macrometa.io";
+const federationUrl = `https://${federationName}`;
 
-  const STREAM_NAME = "api_tutorial_streams";
-  const CONSUMER_NAME = "api_tutorial_streams_consumer";
-  const IS_GLOBAL = true;
+const stream = "api_tutorial_streams";
+const consumerName = "api_tutorial_streams_consumer";
+const isGlobal = true;
 
-  const run = async function () {
-    try {
-      const connection = new APIRequest(FEDERATION_URL);
+const run = async function () {
+  const connection = new APIRequest(federationUrl);
 
-      /* -------------------- Log in (nemo@nautilus.com/xxxxxxx) -------------------- */
+  /* -------------------- Log in (nemo@nautilus.com/xxxxxxx) -------------------- */
```

Comment:
I think I already reviewed this topic in another PR. Please reconcile with @jesusgonzalez12 .